frontend/src/scenes/session-recordings/player/sessionRecordingPlayerLogic.ts

```diff
@@ -621,7 +621,7 @@ export const sessionRecordingPlayerLogic = kea<sessionRecordingPlayerLogicType>(
                 // Check for the "t" search param in the url on first load
                 if (!cache.hasInitialized) {
                     cache.hasInitialized = true
-                    const searchParams = fromParamsGivenUrl(window.location.search)
+                    const searchParams = router.values.searchParams
```

Comment:
`window.location.search` was picking up the storybook iframe searchparams not the applications apparent search params